framework_tvb/tvb/core/services/user_service.py

```diff
@@ -113,9 +111,9 @@ def create_user(self, username=None, password=None, password2=None,
                 admin = admins[randint(0, len(admins) - 1)]
                 if admin.email is not None and (admin.email != TvbProfile.current.web.admin.DEFAULT_ADMIN_EMAIL):
                     # Do not send validation email in case default admin email remained unchanged
-                    email_sender.send(FROM_ADDRESS, admin.email, SUBJECT_REGISTER, admin_msg)
+                    email_sender.send(FROM_ADDRESS, admin.email)
```

Comment:
We should use 4 arguments for the dummy email sender, to behave the same as the original function. Keep these arguments, as they are required by the send function.